server/chat.go

```diff
@@ -12,14 +12,15 @@ import (
 // SendChat is called by the client to send a chat message to the server. The message is then
 // stored in the database.
 func (c *ChatServer) SendChat(ctx context.Context, msg *chat.ChatMessage) (*emptypb.Empty, error) {
+	fmt.Println("chat recieved")
```

Comment:
nit:  look at using some form of structured and leveled logging. 
https://github.com/rs/zerolog
https://github.com/golang/glog